CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # API Mediation Layer Changelog
 
 All notable changes to the Zowe API Mediation Layer package will be documented in this file.
+## `1.19.0`
+
+- Feature: Configure the connection limit for the GW [#843](https://github.com/zowe/api-layer/issues/843)
+- Feature: Minimize size of API Mediation Layer [#909](https://github.com/zowe/api-layer/issues/909)
+- Feature: Allow removal or configuration of a service as a catalog [#727](https://github.com/zowe/api-layer/issues/727)
+- Bugfix: API ID is not sent to Eureka in metadata by Java enabler [#991](https://github.com/zowe/api-layer/issues/991)
+- Feature: Improve connection limits [#987](https://github.com/zowe/api-layer/issues/987)
+- Feature: SAF authorization checking code [#993](https://github.com/zowe/api-layer/issues/993)
```

Comment:
Does the SAF authorization checking code produce the following benefits? If so, which ones?

- speedup conversion (to improve the use of Java reflection)
- allow using classes from multiple modules
- read user's privileges from YML file to simulate verification without ZSS and JZOS library
- calls realized with Java reflection to avoid the dependency to JZOS (outside the mainframe)
- Allows for a ZSS endpoint call to verify the user's access to SAF resource
- Allows a test endpoint (or target endpoint - see bury of profiles) to verify if the feature can block a call

Reply:
Please describe the benefit to the end user.